src/Glue.elm

```diff
@@ -123,10 +126,11 @@ poly :
     , update : subMsg -> subModel -> ( subModel, Cmd msg )
     , subscriptions : subModel -> Sub msg
     }
-    -> Glue model subModel msg subMsg
+    -> Glue model subModel msg subMsg msg
 poly { get, set, init, update, subscriptions } =
     Glue
-        { get = get
+        { msg = identity
```

Comment:
https://github.com/turboMaCk/glue/pull/12/files#diff-6e791af190218f9a5910b36e867fb0a1R46 may help you to understand this. Since `poly` expects that view return parent's `Msg` (`a == msg`) we know this has to be identity function (`a -> a`).